.azure/gpu-tests.yml

```diff
@@ -37,6 +38,21 @@ jobs:
 
     steps:
 
+    - bash: |
+        CHANGED_FILES=$(git diff --name-status master | awk  '{print $2}')
+        FILTER='src/pytorch_lightning|requirements/pytorch|tests/tests_pytorch|examples/pl_'
```

Comment:
do we need to include `setup.py` ?

Reply:
I would keep it as it is right now. setup.py doesn't really influence the tests.